reV/losses/power_curve.py

```diff
@@ -487,8 +536,28 @@ def add_power_curve_losses(self):
             return
 
         wind_resource, weights = self.wind_resource_from_input()
-        pc_losses = PowerCurveLosses(self.input_power_curve,
-                                     wind_resource, weights)
+        power_curve = self.input_power_curve
+
+        if (wind_resource <= power_curve.cutin_wind_speed).all():

```

Comment:
One weird case we are missing here is if there are wind speeds both above the cutoff speed and below the cutin speed. I know this is unlikely, but it would be easy to check for all three cases in one go: 

```
if not power_curve(wind_resource).any():
    msg = ("Input power curve does not generate power for any of "
           "the wind speeds at site {}. No power curve adjustments made!"
           .format(getattr(self, "site", "[unknown]"))
    logger.warning(msg)
    warnings.warn(msg, reVLossesWarning)
    return
```
Not sure what your appetite for making this change is though - I won't impose either way

Reply:
ohhh you're saying if the wind speed timeseries is always low but then has random gusts greater than the cutout? Sounds pretty unlikely haha i say we keep it as is so we get more meaningful logging.